### PR TITLE
Fix ducks and building rotation

### PR DESCRIPTION
**Pull request by JackPlaymint**
This PR rotates the duck roofs on both the factory buildings and the duck counter buildings so that the face the same direction as the building itself.

It also adjusts how dynamically placed buildings are rotated so that they show off both of their detailed sides

### CODE REVIEW

File: frontend/src/components/map/Buildings.tsx [hunk 1]
```diff
@@ -75,7 +75,8 @@ export const Buildings = memo(
                     const coords = getCoords(b.location.tile);
                     const height = getTileHeightFromCoords(coords);
                     const selected = selectedElementID === b.id ? 'outline' : 'none';
-                    const rotation = b.facingDirection == FacingDirectionKind.RIGHT ? -30 : 30;
+                    console.log(b);
```
Comment (chrisfarms) [inline frontend/src/components/map/Buildings.tsx:78]: stray console.log 👁️

Comment (JackPlaymint) [reply frontend/src/components/map/Buildings.tsx]: Oopsy Poopsy!

File: map/ProjectSettings/ScriptableBuildPipeline.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "cacheServerPort": 8126,
+    "threadedArchiving": true,
+    "logCacheMiss": false,
+    "slimWriteResults": true,
+    "maximumCacheSize": 20,
+    "useDetailedBuildLog": false,
+    "useV2Hasher": true,
+    "fileIDHashSeed": 0,
+    "prefabPackedHeaderSize": 2
+}
```
Comment (chrisfarms) [inline map/ProjectSettings/ScriptableBuildPipeline.json:13]: did you mean to commit this? ... seems unrelated

Comment (JackPlaymint) [reply map/ProjectSettings/ScriptableBuildPipeline.json:13]: I did not mean to commit this... I also don't know what it is 🤔. It may be a left over from playing with the rendering. I'll remove it.

Comment (JackPlaymint) [reply map/ProjectSettings/ScriptableBuildPipeline.json:13]: Ooookay... turns out it's auto-generated and will appear after a build forever, so I guess we need it. It's also been added in the Graphics update PR I made.

**Review by chrisfarms — APPROVED** [review]
dunno if that ScriptableBuildPipeline things should be .gitignored or not ... but will leave it up to you

**Review by ldunnplaymint — APPROVED** [review]

**Comment by ldunnplaymint** [comment]
Id gitignore it, easier to add later if its a problem rather than remove it from the list

**Review by ldunnplaymint — APPROVED** [review]
